index.bs

```diff
@@ -3265,8 +3265,9 @@ credential.
 
         When [=clients=], [=client platforms=], or [=authenticators=] display a {{PublicKeyCredentialUserEntity/displayName}}'s value, they should always use UI elements to provide a clear boundary around the displayed value, and not allow overflow into other elements [[css-overflow-3]].
 
-        [=Authenticators=] MUST accept and store a 64-byte minimum length for a {{PublicKeyCredentialUserEntity/displayName}}
-        member's value. Authenticators MAY truncate a {{PublicKeyCredentialUserEntity/displayName}} member's value so that it fits within 64 bytes. See [[#sctn-strings-truncation]] about truncation and other considerations.
+        [=Authenticators=] MUST accept and store a {{PublicKeyCredentialUserEntity/displayName}} member's value
+        whose UTF-8 encoding is 64 bytes or shorter.
```

Comment:
```suggestion
        [=Authenticators=] MUST accept and store a {{PublicKeyCredentialUserEntity/displayName}} member's value
        whose length in the UTF-8 character encoding is 64 bytes or shorter.
```

Reply:
Thanks! I rewrote this a bit, and changed "shorter" to "less" since the subject is now the length rather than the byte string. Also applied the same language to `name` as well.